source/Halibut.OctopusSample/Halibut.OctopusSample.csproj

```diff
@@ -2,22 +2,23 @@
 
   <PropertyGroup>
     <VersionPrefix>0.0.0</VersionPrefix>
-    <TargetFramework>net45</TargetFramework>
     <AssemblyName>Halibut.OctopusSample</AssemblyName>
     <OutputType>Exe</OutputType>
     <PackageId>Halibut.OctopusSample</PackageId>
     <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Include="App.config" />
-  </ItemGroup>
+  <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
+    <TargetFramework>net472</TargetFramework>
```

Comment:
Do we actually need this target at all, netcoreapp2.2 works on Windows?
And I may be mistaken, but won't the net472 pick the halibut.dll from netstandard2 anyway?

Reply:
Alas, this one is a windows form application, which requires the full fat version of .NET.

> but won't the net472 pick the halibut.dll from netstandard2 anyway?
Seems to run on windows OK.

Reply:
Arguably this sample isn't of huge value, it could be removed, but I don't think it is hurting anyone to leave it in.